addons/account/views/bill_preview_template.xml

```diff
@@ -65,7 +65,7 @@
                                             <th name="th_description" class="text-start"><span>Description</span></th>
                                             <th name="th_quantity" class="text-end"><span>Quantity</span></th>
                                             <th name="th_priceunit" class="text-end d-md-table-cell"><span>Unit Price</span></th>
-                                            <th name="th_taxes" class="text-start d-md-table-cell"><span>Taxes</span></th>
+                                            <th name="th_taxes" class="text-end d-md-table-cell"><span>Taxes</span></th>
```

Comment:
If we wanted to have this as a text end shouldn't we want to change the line 83 also ?
Otherwise it will give that if we click on the upload button when having no bills at all
![image](https://github.com/user-attachments/assets/7bbfd9aa-3a38-4765-839d-bf70fe3424aa)